other/enforce-readwriteonce-pod/artifacthub-pkg.yml

```diff
@@ -0,0 +1,29 @@
+name: enforce-readwriteonce-pod
+version: 1.0.0
+displayName: Enforce readwriteoncepod
+createdAt: "2023-12-31T16:10:04.000Z"
+description: >-
+      Some stateful workloads with multiple replicas only allow a single Pod to write
+      to a given volume at a time. Beginning in Kubernetes 1.22 and enabled by default
+      in 1.27, a new setting called ReadWriteOncePod, available
+      for CSI volumes only, allows volumes to be writable from only a single Pod. For more
+      information see the blog https://kubernetes.io/blog/2023/04/20/read-write-once-pod-access-mode-beta/.
+      This policy enforces that the accessModes for a PersistentVolumeClaim be set to ReadWriteOncePod.
+install: |-
+  ```shell
+  kubectl apply -f https://raw.githubusercontent.com/kyverno/policies/main/other/enforce-readwriteonce-pod/enforce-readwriteonce-pod.yaml
+  ```
+keywords:
+  - kyverno
+  - Sample
+readme: |
```

Comment:
Readme is missing the final paragraph common to others. Please look at another sample to copy and paste.